index.js

```diff
@@ -15,22 +15,15 @@ const bubleOpts = {
   }
 }
 
-const shouldSkip = file => contains(file, 'node_modules')
+const transformFile = function (filename, options = {}) {
+  const babelOptions = options.babel ? Object.assign({}, babelOpts, options.babel) : babelOpts
+  const bubleOptions = options.buble ? Object.assign({}, bubleOptions, options.buble) : bubleOpts
 
-const original = require.extensions['.js']
+  const { code: babelCode } = transformFileSync(filename, babelOptions)
```

Comment:
If you would like me to add it to throw our own error message, just let me know what message you would like me to throw :)